src/component/alert/ErrorAlert.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import ErrorAlert from './ErrorAlert';
+
+describe('ErrorAlert component', () => {
+  test('should render without crash', () => {
+    render(<ErrorAlert />);
+  });
+
+  test('should render the message we pass', () => {
+    const message = 'Test error message';
+
+    const { container } = render(<ErrorAlert message={message} />);
+
+    const messageElement = container.querySelector('div');
+    expect(messageElement).toHaveTextContent(message);
```

Comment:
```suggestion
    const { getByText } = render(<ErrorAlert message={message} />);

    getByText(message);
```
Please use [testing-library queries](https://testing-library.com/docs/dom-testing-library/api-queries) here as well, in particular [`getByText`](https://testing-library.com/docs/dom-testing-library/api-queries#bytext), which will throw if it can't find the text content. Try to avoid selecting by DOM element if possible, as it's an implementation detail not visible to end-user, and instead use the query methods which search for page elements in ways that users would.